EOS-uiux-Solutions/user-story
# Fixed Followers view modal.

Signed-off-by: Hrishikesh Agarwal <hrishikeshagarwalv@gmail.com>

**Issue Number**

fixes #213 



**Describe the changes you've made**

I have updated the modal that shows followers. Previously even after voting the user was not showing up in the modal. I have made a state which updates with the response of the API. I have also updated the Graphql query to show the username and profile picture of the user.

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**

NA

**Additional context (OPTIONAL)**

Refactored Code using `useReducer` hook.

**Test plan (OPTIONAL)**

1. Login into the user-story.
2. Check the followers of the story 
3. Vote up a story and see your name added.

**Checklist**
